G1ANT.Addon.IExplorer/G1ANT.Addon.IE.Tests/Commands/ie.getattribute/ShouldGetButtonName_WhenGetAttributeByClass.G1ANT

```diff
@@ -0,0 +1,11 @@
+﻿addon iexplorer version 2.205.17304.1000
+addon core version 4.99.18340.1354
+addon language version 4.99.18340.1354
+
+
+ie.open ‴www.google.pl‴
+ie.getattribute name ‴title‴ search ‴gLFyf gsfi‴ by class result ♥title
+♥testValue1 = ‴Search‴
+♥testValue2 = ‴Szukaj‴
+ie.close
+test condition ⊂♥testValue1== ♥title || ♥testValue2== ♥title⊃
```

Comment:
Please use spaces on both sides of equality operator. This remark applies to all other tests as well.